README.md

```diff
@@ -59,107 +59,117 @@ Click the data migrator to launch it. You will be prompted for a connection to D
-![Generate Export Schema Schema File](images/GenerateExportSchemaSchemaFile.png "Generate Export Schema Schema File")
+3. Select (you may also want to unselect if you are editing your loaded schema) the required entities and attributes combination, (in this example we are creating an xml file for Accounts although you can select more than one entity)
+![Select Schema Configs](images/SelectSchemaConfigs.png "Select Schema Configs")
 
-4.	The next step is to **Save** the schema as shown below:
- ![Generate Export Schema Save](images/GenerateExportSchemaSave.png "Generate Export Schema Save")
+4. To save the file, click on 'Save Schema'
+![Save Schema File](images/SaveSchemaConfigs.png "Save Schema File")
 
-5.	Once the export schema is generated, a "Successfully created XML file" dialog will pop up and the export schema XML file will be generated at the specified location. This file will contain all selected entities and their respective selected attributes and relationships
+Your schema file will be saved at the specified location. This file will contain all selected entities and their respective selected attributes and relationships.
 
- ![Generate Export Schema Save Successful](images/GenerateExportSchemaSaveSuccessful.png "Generate Export Schema Save Successful")
+Below shows an example schema file:
 
- ![Generate Export Schema Output](images/GenerateExportSchemaOutput.png "Generate Export Schema Output")
+![Schema Config Example](images/SchemaConfigExample.png "Schema Config Example")
 
 ### Export Config File
 The CDS data migrator export user interface exposes only a subset of the available configuration points for the data migration operation. Through the provision of an export configuration file, the user can have a fine grain control of the export process. When the export config is not specified then the default settings are applied. While those values are beyond the scope of this documentation, it suffices to say that the default values are expected to satisfy most data operation scenarios.
 
 For more on the data migration config settings [see](https://github.com/Capgemini/xrm-datamigration#Usage)
 
-1.	Select **Generate/Modify Export Config** from the Schema Config tab of the data migrator and then select **Next**
- ![Export Config Start](images/ExportConfigStart.png "Export Config Start")
+1. Select **Data Export** from the top ribbon which will bring you to the below page:
+ ![Initial Export Page](images/InitialExportPage.png "Initial Export Page")
 
-2.	Select the entities and the attributes and then in the **Export Config** browse to a location and input a File Name (in this example we are creating a xml file for Accounts & Contacts), then select **Save**
- ![Export Config Select Entity](images/ExportConfigSelectEntity.png "Export Config Select Entity")
+2. If you have an exisiting export config file, you can click on 'Load' to edit that file. This will pre-input the configs that you had stored in that file.
```

Comment:
```suggestion
2. If you have an existing export config file, you can click on **Load** to edit that file. This will pre-input the configs that you had stored in that file.
```